docs/pywbemcli/commands.rst

```diff
@@ -1534,6 +1548,160 @@ See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
+the Class or property does not exist in the server or the server does not
+allow a client to modify the property.
+
+Once a WBEM server statistics gathering is enabled, an additional column
+``Server Optimes`` appears in the statistics report.
+
+TODO: Example
+
+
+.. index:: pair: statistics commands; statistics server-off
+
+.. _`Statistics server-off command`:
+
+``statistics server-off`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics server-off`` command attempts to disable the statistics gathering
+capability of the currrent WBEM server.  See '_Operation statistics'_ for
+more information on statistics in pywbem/pywbemcli and WBEM servers
+
+Since only some WEBEM server implementation actually implement statistics
+gathering, this request may result in an exception from the server because
+the Class or property does not exist in the server or the server does not
+allow a client to modify the property.
+
+Once a WBEM server statistics gathering is disabled, the  column
+``Server Optimes`` no longer appears in the statistics report.
+
+
+.. index:: pair: statistics commands; statistics status
+
+.. _`Statistics status command`:
+
+``statistics status`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics status`` command attempts display the status of both the
+statistic gathering in the current WBEM server and in pywbemcli.
+
+See '_Operation statistics'_ for
+more information on statistics in pywbem/pywbemcli and WBEM servers
+
+.. index:: pair: statistics commands; statistics reset
+
+.. _`Statistics reset command`:
+
+``statistics reset`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics status`` command attempts display the status of both the
+statistic gathering in the current WBEM server and in pywbemcli.
+
+.. index:: pair: statistics commands; statistics show
+
+.. _`Statistics show command`:
+
+``statistics show`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics show`` command displays the current pywbemcli gathered
+statistics as a table if pywbemcli time statistics is enabled with
+``--timestats`` general options
+
+The following example shows the use of --timestats with the ``cmd`` option in
+the interactive mode executing ``statistics show``.
+
+
+.. code-block:: text
+
+    $ pywbemcli -n mock1 --timestats cmd
+
+    pywbemcli> class enumerate --di --no
+    CIM_Foo
+    CIM_Foo_sub
+    CIM_Foo_sub2
+    CIM_Foo_sub_sub
+     pywbemcli> statistics show
+
+    Statistics: Time(Seconds); Times/lengths: (Avg/Min/Max), single value if all same.
+       Op    Exc     Op Time(S)  Operation
+      Cnt    Cnt    Avg/Min/Max
+    -----  -----  -------------  -------------------
+        1      0          0.000  EnumerateClassNames
+
+
+.. index:: pair: statistics commands; statistics server-stats-show
+
+.. _`Statistics server-stats-show command`:
+
+``statistics server-stats-show`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This command is not currently enabled.
```

Comment:
The data returned is defined by the CIM_CIMOMStatistics.mof which is a subclass of CIM_StatisticalData.  That was implemented by OpenPegasus and the jserver but do not know who else since it never made it into any profiles.

Reply:
DONE, at least the comment about what it does.